pyfolio/utils.py

```diff
@@ -108,29 +96,40 @@ def default_returns_func(symbol):
         Daily returns for the symbol.
          - See full explanation in tears.create_full_tear_sheet (returns).
     """
+    if start is None:
+        start = '1/1/1970'
+    if end is None:
+        end = datetime.now()
```

Comment:
I don't think we should use `datetime.now()` because we only need daily granularity. 
We might want to use `end = pd.Timestamp.now().date()`. 

`pd.io.data.get_data_yahoo` does not return the current trading days data while the market is open, and I'm not sure how long after market close they add the most recent day's data so we can probably leave a one day lag before updating the file.